src/torchmetrics/detection/panoptic_quality.py

```diff
@@ -81,8 +87,8 @@ class PanopticQuality(Metric):
 
     def __init__(
         self,
-        things: Set[int],
-        stuffs: Set[int],
+        things: Collection[int],
```

Comment:
Made the input type generic, I don't see a reason to force the user to pass a `dict` - but we can discuss :)

Reply:
cool :)